articles/azure-monitor/app/opencensus-python.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: OpenCensus Python tracing with Azure Application Insights | Microsoft Docs
-description: Provides instructions to wire up OpenCensus Python tracing with the local forwarder and Application Insights
+title: Monitor Python application with Azure Application Insights | Microsoft Docs
```

Comment:
@mrbullwinkle I need your advise on the service name / branding. Do we want to keep "Application Insights" or we want to call it "Azure Monitor"?